src/v2/BaseLoginRouter.js

```diff
@@ -53,6 +62,7 @@ export default Router.extend({
 
     this.settings = new Settings(_.omit(options, 'el', 'authClient'), { parse: true });
     this.settings.setAuthClient(options.authClient);
+    this.settings.set('proxyIdxResponse', options.proxyIdxResponse);
```

Comment:
you don't have to. line 63 shall take care it.

Reply:
@haishengwu-okta The reason i added this is to get a nested `proxyIdxResponse` object directly. Otherwise it's flattened into each specific property like `proxyIdxResponse.deviceEnrollment.name` or `proxyIdxResponse.deviceEnrollment.platform`. I will need to re-construct the same `proxyIdxResponse` object which is redundant. Do you think there is a better way to handle this?

Reply:
also ^^ @pradeepdewda-okta for the reason i explicitly add it in settings.

Reply:
like @pradeepdewda-okta mentioned, you can add definition to `models/Settings.js`
```
local: {
    ...,
    proxyIdxResponse: 'object',
    ...,
}
```